### PR TITLE
query -> ranking pipeline for 3 models

### PR DESCRIPTION
**Pull request by mmaecki**

### CODE REVIEW

File: scripts/art/query_to_ranking.py [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+            index.add(graph_embeddings)
+
+            model = EmbeddingHead(TrainType.graph)
+            checkpoint = torch.load(models_paths[1], map_location=torch.device('cpu'))
+            model.load_state_dict(checkpoint['state_dict'])
+
+            self.models["graph"] = (model, index)
+
+        if "text+graph" in alg_list:
+            concat_embeddings = self.embeddings["combined_embeddings"].values
```
Comment (SebChw) [inline scripts/art/query_to_ranking.py:56]: This code is almost copied 3 times, you could do a function.

File: scripts/art/query_to_ranking.py [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+
+    def get_query_embedding(self, query):
+        with torch.no_grad():
+            embedding = self.text_embedder.encode(query)
+        return embedding
+    
+
+
+    def search(self, query, topk=10):
+        query = self.clean_text(query)
```
Comment (SebChw) [inline scripts/art/query_to_ranking.py:103]: You shouldn't clean query when you use deep embedders, stopwords are important for them

File: scripts/art/query_to_ranking.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        results_dict = {}
+        for j, model in enumerate(self.models):
+            model, index = self.models[model]
+            model.eval()
+            with torch.no_grad():
+                embedding = model.model_query(query_embedding)
+            embedding = embedding.numpy()
+            embedding = embedding.astype(np.float32)
+            embedding = np.expand_dims(embedding, axis=0)
+            faiss.normalize_L2(embedding) # to chyba nic nie robi gdy jest tylko jeden embedding
```
Comment (SebChw) [inline scripts/art/query_to_ranking.py:115]: a powinno cos robic, bo to normalizuje po dlugosci wektorow

**Review by SebChw — CHANGES_REQUESTED** [review]
